civihr_employee_portal/views/templates/views-view-table--hr-resource-types-list--page.tpl.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file
+ * Template to display a view as a table.
+ *
+ * - $title : The title of this group of rows.  May be empty.
+ * - $header: An array of header labels keyed by field id.
+ * - $caption: The caption for this table. May be empty.
+ * - $header_classes: An array of header classes keyed by field id.
+ * - $fields: An array of CSS IDs to use for each field id.
+ * - $classes: A class or classes to apply to the table, based on settings.
+ * - $row_classes: An array of classes to apply to each row, indexed by row
+ *   number. This matches the index in $rows.
+ * - $rows: An array of row items. Each row is an array of content.
+ *   $rows are keyed by row number, fields within rows are keyed by field ID.
+ * - $field_classes: An array of classes to apply to each field, indexed by
+ *   field id, then row number. This matches the index in $rows.
+ * @ingroup views_templates
+ */
+?>
+
+<div>
+    <a class="btn btn-default btn-primary" href="/admin/structure/taxonomy/hr_resource_type/add">
+        <span class="fa fa-plus"></span> <?php print t('Add HR Resource Type'); ?>
+    </a>
+</div>
+
+<table <?php if ($classes) { print 'class="'. $classes . '" '; } ?><?php print $attributes; ?>>
+  <?php if (!empty($title) || !empty($caption)) : ?>
+    <caption><?php print $caption . $title; ?></caption>
+  <?php endif; ?>
+  <?php if (!empty($header)) : ?>
+    <thead>
+    <tr>
+      <?php foreach ($header as $field => $label): ?>
+        <th <?php if ($header_classes[$field]) { print 'class="'. $header_classes[$field] . '" '; } ?> scope="col">
```

Comment:
I wonder if it wouldn't be better to wrap the if condition in an `empty()` to avoid a `E_NOTICE` in case the index doesn't exist. If you think that it's better, please do the same to the other ifs in this template

Reply:
my mistake, I should added this file in two separate commits to avoid confusion, but this is actually the default view page syntax which is generated by drupal, I only added this div to it https://github.com/compucorp/civihr-employee-portal/blob/47016e5a60831e166d7ab108e078b4d072563d0f/civihr_employee_portal/views/templates/views-view-table--hr-resource-types-list--page.tpl.php#L23

and I would rather not to touch drupal generated syntax unless we discovered there is a reason to do that.